src/openstudio_lib/library/geometry_preview.html

```diff
@@ -2538,6 +2540,7 @@
     } else {
       removeSelection();
     }
+    requestRenderIfNotRequested();
```

Comment:
Do we need to do this on changing render mode, hiding surfaces, or other things too?

Reply:
Interactions on the dat.gui menu trigger an `onChange` event that is captured by `update()`. I added a call to `requestRenderIfNotRequested` there too (see https://github.com/openstudiocoalition/OpenStudioApplication/pull/510/files#diff-574d82d3d9fea756c6e5daf9d279da971c8ad80e9171c4f1ff33de92938dfbfcR3070).

I think it covers the needs, but maybe you think of other interactions that should request rendering?

Reply:
Gotcha!